src/commonMain/kotlin/exchange.dydx.abacus/utils/Logger.kt

```diff
@@ -17,11 +17,19 @@ object Logger {
         }
     }
 
-    fun e(message: () -> String) {
+    fun e(message: () -> String, context: IMap<String, Any>? = null, error: Error? = null) {
```

Comment:
Moving `message` as the last param.  This way, you don't have to change existing calls.

Reply:
sounds good to me!